x-pack/plugins/ml/server/routes/results_service.ts

```diff
@@ -265,4 +283,65 @@ export function resultsServiceRoutes({ router, mlLicense }: RouteInitialization)
       }
     })
   );
+
+  /**
+   * @apiGroup ResultsService
+   *
+   * @api {get} /api/ml/results/:jobId/categorizer_stats
```

Comment:
It requires some brief description for docs to be properly generated, please check other examples

Reply:
Updated here https://github.com/elastic/kibana/pull/74592/commits/a94ecaf246b7f84168eb296b5c2c1a75471353cc